tutorials/docker-compose-on-container-optimized-os.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Running Docker Compose with Docker
+description: Running Docker Compose on a Container-Optimized OS instance.
+author: tswast
+tags: Docker,Docker Compose,Container-Optimized OS,Compute Engine
+date_published: 2017-05-03
+---
+
+This tutorial guides you through running the [Docker
```

Comment:
I like the "in a container" part. I've updated the text.

Not sure about "without installing it". Downloading the image and adding an alias could be considered installing it, just in a different way.